### PR TITLE
$sys_nb_backend_workers removed from local.inc

### PR DESCRIPTION
**Pull request by nterray**
The configuration variable `$sys_nb_backend_workers` is now set via the command `tuleap config-set sys_nb_backend_workers 2`.

Part of [story #35093][0]: execute custom code as artifact post action

[0]: https://tuleap.net/plugins/tracker/?aid=35093

### CODE REVIEW

File: languages/en/installation-guide/step-by-step/redis-configuration.rst [hunk 1]
```diff
@@ -39,7 +39,10 @@ Give it the correct permissions:
     chown codendiadm:codendiadm /etc/tuleap/conf/redis.inc
     chmod 640 /etc/tuleap/conf/redis.inc
 
-In ``/etc/tuleap/conf/local.inc`` you should set ``$sys_nb_backend_workers`` to a number greater than or equal to ``1``.
+You should set ``sys_nb_backend_workers`` to a number greater than or equal to ``1``::
```
Comment (LeSuisse) [inline languages/en/installation-guide/step-by-step/redis-configuration.rst:42]: Maybe we can omit this part? We do the right choice by default and there is not a lot of situation where increasing it is really useful (and it should be done when there is some monitoring in place)

File: languages/en/deployment-guide/15.x.rst [hunk 2]
```diff
@@ -8,7 +8,16 @@ Tuleap 15.5
...
+``$sys_nb_backend_workers`` removed from ``local.inc``
+------------------------------------------------------
+
+The configuration variable ``$sys_nb_backend_workers``,
+used for asynchronous job processing (see :ref:`backend workers guide<installation_redis>`),
+is now set via the following command::
+
+  tuleap config-set sys_nb_backend_workers 2
+
+It is recommended to remove it from ``/etc/tuleap/conf/local.inc`` and issue the command above.
```
Comment (LeSuisse) [inline languages/en/deployment-guide/15.x.rst:20]:
```suggestion
The configuration variable ``$sys_nb_backend_workers``,
used for asynchronous job processing (see :ref:`backend workers guide<installation_redis>`),
is now set via ``tuleap config-set``.

It is recommended to remove it from ``/etc/tuleap/conf/local.inc``. If you had set it specifically to a value greater than 2, you can keep it your setting by issuing the following command:

.. sourcecode:: shell

    tuleap config-set sys_nb_backend_workers <NB>
  
```

We probably should not encourage operators to specifically override our default values if it is not necessary. This give us the possibility to adjust in the future if we determine than something else is better and to propagate the better setting to more instances.

File: languages/en/installation-guide/step-by-step/redis-configuration.rst (hunk 1 above)
Comment (nterray) [reply languages/en/installation-guide/step-by-step/redis-configuration.rst]: Ok I remove the sentence.

File: languages/en/deployment-guide/15.x.rst (hunk 2 above)
Comment (nterray) [reply languages/en/deployment-guide/15.x.rst]: Ok

**Pull request merged**